zebedee-cms/src/test/java/com/github/onsdigital/zebedee/api/ContentTest.java

```diff
@@ -108,7 +116,7 @@ public void setUp() throws Exception {
     }
 
     @Test
-    public void writeVersionFileV1() throws Exception {
+    public void ResponseContainsFileDownloadLinkWhenJSONV1IsWritten() throws Exception {
```

Comment:
Still not following Java standards, it should start with lowercase
```suggestion
    public void responseContainsFileDownloadLinkWhenJSONV1IsWritten() throws Exception {
```

Reply:
sorry, my mistake I missed the other one...